### PR TITLE
Changed description text for Stock Round.

### PR DESCRIPTION
**Pull request by ryanespin**
Addresses Issue #661

### CODE REVIEW

**Comment by tobymao** [comment]
this is not correct for 1889 since you sell buy or buy sell

**Comment by ryanespin** [comment]
Ah okay, let me change it to be configurable.  Thanks!

**Comment by jenf** [comment]
Use sell_buy_order variable

**Comment by ryanespin** [comment]
@jenf, used the variable.  Not sure if this is the most optimal way though.

File: lib/engine/round/stock.rb [hunk 1]
```diff
@@ -32,7 +32,14 @@ def name
       end
 
       def description
-        'Buy and Sell Shares'
+        case @sell_buy_order
+        when :sell_buy_or_buy_sell
+          'Buy and Sell Shares'
```
Comment (tobymao) [inline lib/engine/round/stock.rb:37]: how about 

Buy or Sell Shares
Sell Then Buy Shares
Sell then Buy then Sell Shares
Sell Buy, Sell Shares

**Pull request merged**